### DISCUSSION TITLE
A bunch of ideas, mostly UI/UX

### DISCUSSION THREAD

**Discussion by mocmocmoc** [Nevysha/Cozy-Nest#132] [Ideas]
Before
![cozy_before](https://github.com/Nevysha/Cozy-Nest/assets/6404011/9c6f7aac-1a5a-459a-adb1-0204527a037d)

After
![cozy_after](https://github.com/Nevysha/Cozy-Nest/assets/6404011/c71f95d0-d0cd-4c46-8ef5-2cbe08b24486)
![cozy_after2](https://github.com/Nevysha/Cozy-Nest/assets/6404011/d472f1a8-7d50-49ad-89bb-e7978290a1c2)

Just bored and made a wish list. Really hope you can declutter the extension menus with accented colored lines, having a bunch of them is a pain to navigate. Love your work!

**Comment by Nevysha** [comment]
Those are really good idea !
FYI : I'm already working on a colored prompt ;) . Here's a prototype I made : 
![Cozy-Prompt-test](https://github.com/Nevysha/Cozy-Nest/assets/122687716/1d57178e-b967-4a72-af81-052aacdff0c2)

This was just to think about this functionnality. I'll probably use ace lib to do it. In the next version I'm integrating the base for using ace on another part of the app : Exif Editor
![image](https://github.com/Nevysha/Cozy-Nest/assets/122687716/cfb1b64c-083b-4b89-8823-1c8a06bd0718)

**Comment by That-Dude** [comment]
I came here to suggest mostly the same ideas :-)

Incredible work @Nevysha I've been using this for 2 days and can never go back to the default theme.

**Comment by Nevysha** [comment]
I'm working on it for the next update (which take time to ship, got a lot of work on other project than Cozy Nest lately...)

https://github.com/Nevysha/Cozy-Nest/assets/122687716/fbd00319-355a-46eb-bd04-e4b0eccbfd78

![image](https://github.com/Nevysha/Cozy-Nest/assets/122687716/2924b8e5-0fca-434c-9770-1e5dff38e7ce)
![image](https://github.com/Nevysha/Cozy-Nest/assets/122687716/6c7899aa-f1ea-49e5-aacc-ac98b4f2d6f1)
